pkg/istio/wasm.go

```diff
@@ -67,12 +69,12 @@ func PluginPolicyFromRateLimitPolicy(rlp *apimv1alpha1.RateLimitPolicy, pluginSt
 	return pluginPolicy
 }
 
-func MergeMapStringPluginPolicy(modified *bool, existing *map[string]PluginPolicy, desired map[string]PluginPolicy) {
```

Comment:
why desired should be a pointer to a map?

Reply:
I believe the whole map shouldn't be copied, right?

Reply:
TIL